apps/editor/src/app/utility.ts

```diff
@@ -8,7 +8,8 @@ import {ElementID} from '../shared/elementID'
 import {ClientSettings} from '../shared/types'
 
 export enum LocalStorageKey {
-  SessionToken = 'sessionToken'
+  SessionToken = 'sessionToken',
+  isGridView = 'isGrid'
```

Comment:
We probably want a more descriptive and specific key. Like `ImageListLayout` which is a enum/string